test/test_common/environment.cc

```diff
@@ -401,6 +401,8 @@ Json::ObjectSharedPtr TestEnvironment::jsonLoadFromString(const std::string& jso
   return Json::Factory::loadFromString(substitute(json, version));
 }
 
+// This function is not used for Envoy Mobile tests, and ::system() is not supported on iOS.
```

Comment:
Since we have a comment about this in the .cc file, I don't think we need to repeat it here, so let's remove it.